### ISSUE TITLE
Possible regression of https://github.com/react-boilerplate/react-boilerplate/issues/2192

### ISSUE THREAD

**Issue by joe-fenton** [react-boilerplate/react-boilerplate#2441]
## Issue Type

- [x] Bug (https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md#bug-reports)
- [ ] Feature (https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md#feature-requests)

## Description

I have the same bug that is described in issue #2192 which I think is supposed to be fixed in v3.6. I am using v3.7 and get the same errors after running setup. I can't then clean the example project as it has failed to create a git repo. This is using the terminal in Visual Studio Code on Windows 10.

## Steps to Reproduce

Exactly as in #2192 

````
git clone --depth=1 https://github.com/react-boilerplate/react-boilerplate.git

npm run setup
- or -
yarn setup

(when asked if you want to clear the old repo, say "Y")
````
following this yarn run clean fails with:
```
Cleanup started... ✓fatal: Not a git repository (or any of the parent directories): .git
exec: fatal: Not a git repository (or any of the parent directories): .git
```

# Versions

- React-Boilerplate (see `package.json`): 3.7.0
- Node/NPM: 8.12.0
- Browser: Chrome

**Comment by joe-fenton** [comment]
What is going on with the bot here? I've filled the issue out? @julienben

**Comment by julienben** [comment]
It looks for these strings in your issue:
```yaml
content:
  - "Issue Type"
  - "Description"
  - "Steps to Reproduce"
  - "Versions"
```
They're all there but maybe it's case-sensitive and doesn't work with "Steps to reproduce".

I'll edit your Issue, reopen and see if it doesn't try to close it again.

**Comment by joe-fenton** [comment]
Yes I'm on windows 10. I put that in my original issue but took it out in case I was breaking the template

**Comment by julienben** [comment]
I'm afraid I can't reproduce your issue on my end. (I don't have a PC and the setup script works on my mac.)

Your best chance at debugging this is to carefully step through ```setup.js``` and see at what point it stops behaving as expected.

Or maybe someone else with a Windows env can try and see if they get the same issue? @Mensae or @SVAR7BERG or @mciastek?

**Comment by Mensae** [comment]
I have to confirm that the issue is either back or the previous solution is not consistent.

I'll look into it.

**Comment by Mensae** [comment]
@joe-fenton Can you check if [this](https://github.com/react-boilerplate/react-boilerplate/pull/2478) works for you?

**Comment by julienben** [comment]
@joe-fenton, it would indeed be super helpful if you can try out @Mensae 's fix on your setup.

I've opened a custom react-boilerplate branch with the fix merged into it so it's easier for you to try it out: https://github.com/react-boilerplate/react-boilerplate/tree/setup-fix-for-windows

**Comment by joe-fenton** [comment]
Sorry for the delay. I've just checked and cloned that branch. It worked fine!
